app/assets/stylesheets/mtl/layouts/_default.scss

```diff
@@ -21,6 +21,10 @@
   background-color: $mtl-layout-default-header-bg;
   @include clearafter;
 
+  * {
```

Comment:
uh... can we avoid things like that by chance, really not a fan of using `*`, as this would mess up, e.g. the following:

```
<div class="grey-color">asdsa <span>didum</span></div>
```

isn't simply setting color: enough + a few specific overrides for links?